contributing/code/tests.rst

```diff
@@ -59,6 +62,62 @@ what's going on and if the tests are broken because of the new code.
 
         $ php ./phpunit src/Symfony/Component/Finder/
 
+Through Docker
+--------------
+
+You must `install Composer`_ and ``docker``.
```

Comment:
I will add something like:
Starting from v6.2, you can execute phpunit symfony tests inside docker with the folowing steps

Reply:
The PR targets 6.2 so it won't appear on previous versions.